src/api/app/policies/webui/token_policy.rb

```diff
@@ -0,0 +1,23 @@
+class Webui::TokenPolicy < ApplicationPolicy
+  class Scope < Scope
+    def initialize(user, scope)
+      raise Pundit::NotAuthorizedError, reason: ApplicationPolicy::ANONYMOUS_USER if user.nil? || user.is_nobody?
+
+      super(user, scope)
+    end
+
+    def resolve
+      scope.where(user: user).where.not(type: 'Token::Rss').includes([:package])
+    end
+  end
+
+  def create?
+    # TODO: when user_profile_redesign is rolled out, remove the Flipper check
+    Flipper.enabled?(:user_profile_redesign, user)
```

Comment:
We should also check if the token to be created is assigned to the user, no?  

```suggestion
    record.user == user && Flipper.enabled?(:user_profile_redesign, user)
```

Reply:
Fixed.